client/src/utils/googleAuth.js

```diff
@@ -0,0 +1,13 @@
+import axios from "axios";
+
+const userAPIpath = "http://localhost:3001/api/authentication/google";
```

Comment:
```suggestion
const userAPIpath = "/api/authentication/google";
```
It looks like this is your problem. If you also change the proxy to "http:" instead of "https:" in the package.json then you should be good to go

Reply:
@kozaktar  Did this work?

Reply:
No I was getting an error.  I used fetch and that seems to work